frontend/src/metabase/auth/components/Login/Login.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { createStyles } from "@mantine/core";
+
+export const useStyles = createStyles(theme => ({
+  title: {
+    color: theme.colors.text[2],
+    fontSize: "1.25rem",
+    fontWeight: "bold",
+    lineHeight: "1.5rem",
+    textAlign: "center",
+  },
+  panel: {
```

Comment:
Could we not use `Box` with a `m` prop for this purpose since we're just changing spacing here? I think that's the eventual "right" way to handle a case like this.